ext/afform/core/Civi/Api4/Action/Afform/Submit.php

```diff
@@ -82,6 +91,21 @@ protected function processForm() {
     ];
   }
 
```

Comment:
/**
   * Recursively removes 'joins' from $submissionData in favor of submiting only '_joins'
   */
  private function removeJoinsElement(&$submissionData) {
    foreach ($submissionData as $key => &$value) {
      if ($key === 'joins') {
        unset($submissionData[$key]);
      }
      elseif (is_array($value)) {
        $this->removeJoinsElement($value);
      }
    }
    return $submissionData;
  }